front/src/Phaser/Game/GameScene.ts

```diff
@@ -127,6 +130,8 @@ export class GameScene extends ResizableScene implements CenterListener {
     // A promise that will resolve when the "create" method is called (signaling loading is ended)
     private createPromise: Promise<void>;
     private createPromiseResolve!: (value?: void | PromiseLike<void>) => void;
+    private helpCameraSettingsElement!: Phaser.GameObjects.DOMElement; //#700
+    private helpCameraSettingsOpened: boolean = false; //#700
```

Comment:
We should move all this code in a complete separate Scene.

- Create the new class (copy paste from MenuScene)
- Reference the new scene in index.ts
- Start the new scene when the menuScene is started